src/modules/home.js

```diff
@@ -0,0 +1,12 @@
+function home() {
+  const element = document.createElement('div');
+
+  element.innerHTML = `<div class="main-wrap first-page">
+              <p class="main-text">Wellcome to the </p><br>
+              <h1>"our restaurant"</our></h1>
+              </div> `;
+
+  return element;
+}
+
```

Comment:
- [ ] Rather than using a regular function please use the es6 arrow function. Please adjust other affected modules.